### DISCUSSION TITLE
What is loadbalancer policy for stateful sctp connections

### DISCUSSION THREAD

**Discussion by Tanvi-1** [loxilb-io/loxilb#353] [Q&A] [answered]
In case of 5G, there is sctp connection between RAN and AMF, how loxilb is distributing UE across AMF? what state is maintained for each ue, in which file the policies are written?

**Comment by UltraInstinct14** [comment]
Support is planned for NGAP. Based on that such distribution would be possible. Currently, sctp session L4 LB is supported. Further, fine grained decisions can be made after NGAP awareness is implemented.

**Comment by TrekkieCoder** [comment, marked as answer]
We are finally here with support of stateful sctp LB. Please find the related info in this [blog](https://www.loxilb.io/post/ngap-load-balancing-with-loxilb).
